lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,51 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=6, lat_n=6, lon_max=180,
```

Comment:
>I'm now inclined to suggest simplifying the arguments to...

Yeah that makes sense - gives a more transparent interface, and more flexibility. Did you mean to change the radius to 5000?

>Maybe because the default globe is different between Cartopy and the Geodesic code? I think it should be WGS84.

I hadn't considered that, although I would be surprised if that was the case as I definitely used the WGS84 parameters in geodesic. Does cartopy definitely use this?